.eslintrc.json

```diff
@@ -181,11 +177,20 @@
         "no-useless-constructor": "off",
         "no-underscore-dangle": "off",
         "jsdoc/newline-after-description": "off",
+        "no-console": [
+          "error",
+          {
+            "allow": [
```

Comment:
Are you sure we want this? We might be better off leaving it to loggerService.

Reply:
This already exists. See above. I think the tool has re-arranged the rules.

Reply:
Ah, fair enough.